distribution_circuits_sale/models/sale.py

```diff
@@ -128,6 +128,12 @@ def _compute_amount_due(self):
             for invoice in open_invoices:
```

Comment:
the open invoice are computed inside the partner.credit field which is computed on the unmatch account.move.line it is the standard in Odoo. 

So you should remove the computation on open_invoice as it will lead to problem to compute customer credit.

Reply:
Okay, I didn't know that. 
And anyway, after testing it the right way, the customer credit does take into account open invoices. So this change is useless.